packages/sanity/src/studio/screens/CorsOriginErrorScreen.tsx

```diff
@@ -0,0 +1,61 @@
+import {Card, Dialog, Stack, Button, Text} from '@sanity/ui'
+import React, {useCallback} from 'react'
+
+interface CorsOriginErrorScreenProps {
+  projectId?: string
+}
+
+export function CorsOriginErrorScreen(props: CorsOriginErrorScreenProps) {
+  const {projectId} = props
+
+  const corsUrl = `https://sanity.io/manage/project/${projectId}/settings/api`
+
+  const origin =
+    window.location.origin ||
+    window.location.href.replace(new RegExp(`${window.location.pathname}$`), '')
```

Comment:
Under what circumstances (if any?) will `window.location.origin` not be defined?

Reply:
I think this is wrong by me – it should be `window.location.origin` only